packages/truffle-contract/lib/reformat.js

```diff
@@ -37,14 +37,14 @@ const _convertNumberArray = function(arr, format){
  *   - object (with index keys and optionally, named keys)
  *   - array
  *   - single primitive
- * @param  {String|Object|Array} result  web3 call result
- * @param  {Array}               outputs ABI outputs array
+ * @param  {String|Object|Array} result      web3 call result
+ * @param  {Array}               abiSegment  event inputs OR .call outputs
```

Comment:
I think these are called `parameters` in the ABI spec

Reply:
Why are they so normie over there?

Reply:
Ugh, normies!